lib/raster/mask_info.c

```diff
@@ -53,16 +49,73 @@ char *Rast_mask_info(void)
     return G_store(text);
 }
 
+/**
+ * @brief Get raster mask status information
+ *
+ * @param[out] name Name of the raster map used as mask
+ * @param[out] mapset Mapset name of the map
+ * @param[out] is_mask_reclass True, if MASK raster is a reclass
```

Comment:
missing parameters

Reply:
Document that passing NULL to one of the reclass parameters will skip the reclass output.